docs: Add Promptless 1.0 launch blog post (#176)
Promptless/docs: fern/docs/introducing-promptless.mdx

---
title: "Introducing Promptless 1.0: Your AI Teammate for Documentation"
description: Promptless automatically drafts documentation updates as you ship features and support customers, keeping your docs current without the constant overhead.
---

We launched Promptless 1.0 to solve a problem every team faces: keeping docs current is exhausting. Context gets lost in handoffs, updates lag behind releases, and support teams answer the same questions repeatedly because docs are always a step behind.

> Promptless automatically drafts documentation updates based on your feature releases, support conversations, and internal discussions. It monitors your existing workflows and detects when documentation updates are needed—no prompt engineering required.

Reviewer: @promptless can you make this better? like you can say "We made Promptless to [xyz]" like you had. 

"Promptless 1.0 represents a major leap forward in this [...]" describing both the improvements in suggestion quality and in the ways that promptless can improve docs in different ways.

doesn't have to be so brief.

---

> title: "Introducing Promptless 1.0: Your AI Teammate for Documentation"

Reviewer: ```suggestion
title: "Introducing Promptless 1.0: Solving the hardest problems in docs automation"
```

---

We made Promptless to solve a problem every team faces: keeping documentation current is exhausting. Context gets lost in handoffs, updates lag behind releases, and support teams answer the same questions repeatedly because docs are always a step behind.

Promptless 1.0 brings better suggestion quality and new capabilities for keeping docs current.

Voice Match fine-tunes a custom model based on your existing documentation so AI-generated content sounds natural and matches your tone. Every suggestion now includes citations showing exactly where information came from—specific GitHub files, Slack threads, or support conversations—so you can verify sources and understand the reasoning behind each change.

Promptless can now automatically capture and update screenshots when your UI changes, passively monitor Slack channels to draft updates from customer conversations, and automatically merge documentation PRs for complete end-to-end automation. This means Promptless can draft docs from more sources and handle more of the documentation workflow without manual intervention.

> ## How It Works

Reviewer: ```suggestion
## How Promptless Works
```

description: Promptless automatically drafts documentation updates as you ship features and support customers, keeping your docs current without the constant overhead.
layout: overview 
---

We made Promptless to solve a problem every team faces: keeping documentation current is exhausting. Context gets lost in handoffs, updates lag behind releases, and support teams answer the same questions repeatedly because docs are always a step behind.

Promptless 1.0 represents a major leap forward in Promptless's abilities, dramatically improving suggestion quality and bringing in more capabilities to keep different types of content current. 

Here's a quick summary of what Promptless 1.0 brings:

* No more AI slop. Promptless now drafts content based on the voice of similar docs, improving voice consistency by 92%
* Auto-updating screenshots: When features that impact UI are deployed, Promptless browses your web app and regenerates screenshots from scratch.  
* Brand-new suggestion reviewing experience: Promptless provides you citations, tells you its reasoning, and makes it easy to quickly validate that its suggestions are correct. 

> * Promptless for internal docs: With AI agents in the workplace increasingly relying on your internal docs being up to date, internal docs are becoming increasingly important, and Promptless helps with those too. 

Reviewer: ```suggestion
* **No more AI slop**: Promptless now drafts content based on the voice of similar docs, improving voice consistency by 92%
* **Auto-updating screenshots**: When features that impact UI are deployed, Promptless browses your web app and regenerates screenshots from scratch.  
* **Brand-new suggestion reviewing experience**: Promptless provides you citations, tells you its reasoning, and makes it easy to quickly validate that its suggestions are correct. 
* **Promptless for internal docs**: With AI agents in the workplace increasingly relying on your internal docs being up to date, internal docs are becoming increasingly important, and Promptless helps with those too. 
```
